include/swift/AST/DiagnosticsSema.def

```diff
@@ -5243,6 +5243,15 @@ NOTE(actor_mutable_state,none,
 WARNING(shared_mutable_state_access,none,
         "reference to %kind0 is not concurrency-safe because it involves "
         "shared mutable state", (const ValueDecl *))
+ERROR(shared_mutable_state_decl,none,
+      "%kind0 is not concurrency-safe because it is global shared mutable "
+      "state", (const ValueDecl *))
+NOTE(shared_mutable_state_decl_note,none,
+     "isolate %0 to a global actor, or convert it to a constant",
```

Comment:
Would it make sense to be "or change it to a 'let' constant"?
Just asking because that is a terminology we already use in other diagnostic https://github.com/apple/swift/blob/db22b88af94e6a8f68b4133b244f095395ac9660/include/swift/AST/DiagnosticsSema.def#L6610C1-L6610C1